slidingwindow/MinimumSubarrayWithTarget.py

```diff
@@ -0,0 +1,18 @@
+import math
+class Solution:
+    def minSubArrayLen(self, target: int, nums: List[int]) -> int:

```

Comment:
Don't copy Leetcode solution over here. Give this code its own main method and it should be executable.
Please refer other code written by me.